ignite/pkg/xhttp/response.go

```diff
@@ -3,16 +3,21 @@ package xhttp
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"net/http"
 )
 
 // ResponseJSON writes a JSON response to w by using status as http status and data
 // as payload.
 func ResponseJSON(w http.ResponseWriter, status int, data interface{}) error {
+	var errMarhsal error
 	bdata, err := json.Marshal(data)
 	if err != nil {
 		status = http.StatusInternalServerError
-		bdata, _ = json.Marshal(NewErrorResponse(errors.New(http.StatusText(status))))
+		bdata, errMarhsal = json.Marshal(NewErrorResponse(errors.New(http.StatusText(status))))
```

Comment:
This one can be annoying, because I want to be able to ignore an error.

Is there a way to disable it from code (via a comment?).

Reply:
```go
bdata, _ = json.Marshal(NewErrorResponse(errors.New(http.StatusText(status)))) //nolint:errchkjson
```

Reply:
ah ok it's related to json, fine!